brian2/stateupdaters/base.py

```diff
@@ -127,10 +138,16 @@ def apply_stateupdater(equations, variables, method):
             if the_method is None:
                 raise ValueError(('No stateupdater that is suitable for the '
                                   'given equations has been found.'))
-            logger.info('Using state updater: %r' % the_method)
+            if group_name is not None:
+                msg_text = ("No numerical integration method specified for group "
+                            "'{group_name}', choosing method '{method}'.")
+            else:
+                msg_text = ("No numerical integration method specified, "
+                            "choosing method '{method}'.")
+            logger.info(msg_text.format(group_name=group_name,
+                                        method=the_method), 'method_choice')
```

Comment:
Ah, ok. Well, I think either all the state updater messages should go to debug or all should stay at info, most of the time the user does not care about the integration method and we have this other issue open about warnings for instabilities. Actually, if we move them to debug, then nothing remains at the info level and we wouldn't need a new log level in the first place... We could decide to lower some messages from warning to info to make them sound less severe (e.g. everything related to performance?).